financial-connections/src/main/java/com/stripe/android/financialconnections/ui/theme/Theme.kt

```diff
@@ -0,0 +1,146 @@
+package com.stripe.android.financialconnections.ui.theme
+
+import androidx.compose.material.Colors
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+private val LightColorPalette = FinancialConnectionsColors(
```

Comment:
why not extend material theme like we did for Payments? 

One advantage is that we can then potentially use any component you build elsewhere in the SDK. Inside a component you could use `color = MaterialTheme.colors.surface` rather than `FinancialConnectionsTheme.colors.textBrand` and for any color that doesn't match with material theme you can define a custom color and call it like `MaterialTheme.FinancialConnectionsColors.myDopeColor`. I think we could also drop the debug colors if we do this. What are you thoughts? 

The payments theme is a little confusing because of the theme work I did, but I think Link has a great example of what I'm describing: https://github.com/stripe/stripe-android/blob/master/link/src/main/java/com/stripe/android/link/theme/Theme.kt

Reply:
I initially thought about this. So far for our usecase I found more advantages on speaking the exact same language as the design team does when it comes to typography and colors.

Regarding typography, Material and Financial Connections theming differs: Material defines h1-h6, subtitles 1-2, bodies, etc. We define regular and emphasized titles, subtitles and bodies. Extending would make these two theming languages to coexist, probably leading to confusion in the future (i.e: _should I use material or the extended theming for this new component?_)

Kind of the same thing applies to colors. As an example, Material defines primary and secondary colors, an a unique text color over surface. On our coloring, we define primary and secondary coloring that paint over the same surface, not supported by Material.

Overall, rather than having to constantly having to translate Financial Connections theming to Material, I found having a single source of truth on our theming definition, and customizing components accordingly based on our guidelines more useful than two coexisting themes.

But happy to discuss this more deeply!

Reply:
My thought is that an android dev is more used to the normal material naming rather than the naming from design. 

but.... then a new dev would have to translate anyways from design to material. SO I think both solutions have a translation somewhere. I think for my usecase in payments I needed to make something both externally facing and something that we could use generically for future payments UIs. Here I don't think we have those restrictions and your implementation is optimal. 

I don't think this is a trapdoor anyways. I don't want to block you any further :D